docs/configuration.md

```diff
@@ -194,3 +194,21 @@ const awsRum: AwsRum = new AwsRum(
 | Name | Type | Default | Description |
 | --- | --- | --- | --- |
 | eventLimit | Number | `10` | The maximum number of resources to record load timing. <br/><br/>There may be many similar resources on a page (e.g., images) and recording all resources may add expense without adding value. The web client records all HTML files and JavaScript files, while recording a sample of stylesheets, images and fonts. Increasing the event limit increases the maximum number of sampled resources. |
+| ignore | Function(event: PerformanceEntry) : any | `(entry: PerformanceEntry) => !/^https?:/.test(entry.name)` | A function which accepts a [PerformanceEntry](https://developer.mozilla.org/en-US/docs/Web/API/PerformanceEntry) and returns a value that coerces to true when the PerformanceEntry should be ignored.</br></br> By default, [PerformanceResourceTiming](https://developer.mozilla.org/en-US/docs/Web/API/PerformanceResourceTiming) entries with URLs that do not have http(s) schemes are ignored. This causes resources loaded by browser extensions to be ignored. |
+
+For example, the following telemetry config array causes the web client to ignore all resource entries.
```

Comment:
nit: As thought exercise, any other common ways customers might use ignore? If yes, we can add examples along that.

Reply:
I had some other examples, such as blocking a [hostname](https://github.com/aws-observability/aws-rum-web/pull/419/commits/e44c80346ea12132bf12b3cce6d5eed4fef28a3d#diff-17ed18489a956f326ec0fe4040850c5bc9261d4631fb42da4c52891d74a59180R149-R156) or [type casting `entry` into other subclasses](https://github.com/aws-observability/aws-rum-web/pull/419/commits/e44c80346ea12132bf12b3cce6d5eed4fef28a3d#diff-17ed18489a956f326ec0fe4040850c5bc9261d4631fb42da4c52891d74a59180R149-R156), but I was asked to remove them

Reply:
maybe we can consider in future iterations of doc updates/improvements based on customer experience with this

Reply:
yes absolutely